### ISSUE TITLE
factor out shapes and buttons for Play/Pause/Stop

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/scenery-phet#428]
In Gas Properties, the design references TimerNode and PeriodTimerNode for 2 of the meters that could be reused or referenced.  Both have a 'play' button, and it sure looks like code was copied from one to the other:

SCENERY_PHET/TimerNode:

```js
    var playPauseHeight = resetAllShape.bounds.height;
    var playPauseWidth = playPauseHeight;
    var halfPlayStroke = 0.05 * playPauseWidth;
    var playOffset = 0.15 * playPauseWidth;
    var playShape = new Shape().moveTo( playPauseWidth - halfPlayStroke * 0.5 - playOffset, 0 )
      .lineTo( halfPlayStroke * 1.5 + playOffset, playPauseHeight / 2 - halfPlayStroke - playOffset )
      .lineTo( halfPlayStroke * 1.5 + playOffset, -playPauseHeight / 2 + halfPlayStroke + playOffset )
      .close()
      .getOffsetShape( -playOffset );
```

PENDULUM_LAB/PeriodTimerNode:

```js
    var playPauseSize = uArrowShape.bounds.height;
    var halfPlayStroke = 0.05 * playPauseSize;
    var playOffset = 0.15 * playPauseSize;
    var playShape = new Shape().moveTo( playPauseSize - halfPlayStroke * 0.5 - playOffset, 0 )
      .lineTo( halfPlayStroke * 1.5 + playOffset, playPauseSize / 2 - halfPlayStroke - playOffset )
      .lineTo( halfPlayStroke * 1.5 + playOffset, -playPauseSize / 2 + halfPlayStroke + playOffset )
      .close()
      .getOffsetShape( -playOffset );
```

(For the record, PeriodTimerNode appears to have been copied from TimerNode by MLL.)

The above should be factored out into something like `PlayShape`, similar to `UTurnArrowShape`.

And I'll need to use this shape in Gas Properties' 'Collision Counter'.

@ariel-phet FYI I'm going to do this.

**Comment by pixelzoom** [comment]
@samreid @jonathanolson FYI, this is going to require changes to TimerNode and PeriodTimerNode.

**Comment by pixelzoom** [comment]
On hold until we decide what the icons should be in https://github.com/phetsims/scenery-phet/issues/430.

**Comment by pixelzoom** [comment]
I made an attempt at this and failed.  I'm going to move forward by creating my own button for the Collision Counter in Gas Properties.   If/when someone factors out reusable shapes and a button, I'll be happy to use it in Gas Properties.

Assigning to @ariel-phet to prioritize and assign.

**Comment by pixelzoom** [comment]
Btw... We could use the same 'play' shape everywhere and still differentiate between TimerNode and PlayPauseButton by varying the fill.    I have a hard time believing that there would be any confusion between these:

![screenshot_880](https://user-images.githubusercontent.com/3046552/48737285-e0ca9100-ec0a-11e8-9693-b6fc3dfafb1a.png)    ![screenshot_879](https://user-images.githubusercontent.com/3046552/48737295-e3c58180-ec0a-11e8-8f8a-1ed9368e1a03.png)

**Comment by pixelzoom** [comment]
To continue work on Gas Properties, I've created yet-another play/reset button for the Collision Counter. See `PlayResetButton`:

![screenshot_882](https://user-images.githubusercontent.com/3046552/48743521-b4217400-ec20-11e8-9deb-b11cdf0d0fd5.png)  ![screenshot_883](https://user-images.githubusercontent.com/3046552/48743523-b552a100-ec20-11e8-9101-c15bab03ed07.png)

**Comment by ariel-phet** [comment]
@pixelzoom I like your suggestion of the alternate fill. I am also OK if you want to change timer node to use the pause icon. Feel free to make these changes at your convenience. For timer node, I believe @Denz1994 was doing some work, so you can assign changes for that component to him.

**Comment by pixelzoom** [comment]
Shapes are committed.

I went ahead and did the TimerNode work in https://github.com/phetsims/scenery-phet/issues/430, since I needed someplace to test the button icons.   I also used the shapes in PlayPauseButton, and verified that there's no change to that button's appearance.

PeriodTimerNode in pendulum-lab should be (or rather should have been) replaced with TimerNode.  Tracking that in https://github.com/phetsims/pendulum-lab/issues/207.

Closing.
